screens/MerchantScreens/CreateMerchantScreen.tsx

```diff
@@ -14,17 +14,20 @@ import { useRefresh } from "../../hooks/useRefresh";
 import AsyncStorage from "@react-native-async-storage/async-storage";
 
 export default function CreateMerchant({ navigation }: RootStackScreenProps<'CreateMerchant'>) {
-
-    const passwordHash = useAuth();
+    const passwordHash = useAuth({
+        onRetrieved: (passwordHash) => {
+            getCategories({ variables: { passwordHash } });
```

Comment:
Another use before definition

Reply:
In JS there is a concept called hoisting. Declarations can be used in earlier lines of code without error.

Reply:
very interesting